src/ggml.c

```diff
@@ -8446,7 +8469,7 @@ static void ggml_compute_forward_concat_f32(
     GGML_ASSERT(nb10 == sizeof(float));
 
     for (int i3 = 0; i3 < ne3; i3++) {
-        for (int i2 = ith; i2 < ne2; i2++) {
+        for (int i2 = ith; i2 < ne2; i2 += nth) {
```

Comment:
@FSSRepo FYI there was a bug in `ggml_concat`

Reply:
This already solves it anyway.

Reply:
Yes - waiting for CI to pass and will merge